app/bundles/LeadBundle/LeadEvents.php

```diff
@@ -343,4 +343,14 @@
      * @var string
      */
     const COMPANY_POST_DELETE = 'mautic.company_post_delete';
+
+
+    /**
+     * The mautic.remove_do_no_contact event is dispatched when a new submission is fired.
+     *
+     * The event listener receives a Mautic\FormBundle\Event\SubmissionEvent instance.
+     *
+     * @var string
+     */
+    const REMOVE_DO_NO_CONTACT = 'mautic.remove_do_no_contact';
```

Comment:
Can we name this 

```
const FORM_SUBMIT_REMOVE_DO_NO_CONTACT = 'mautic.form_submit_remove_do_no_contact';
```

I just don't want it to be confused with the potential future event if we dispatch something global for when a DNC status is removed.